app/controllers/SingleRecordController.scala

```diff
@@ -52,6 +52,14 @@ class SingleRecordController @Inject() (
     (identify andThen profileAuth andThen getData andThen requireData).async { implicit request =>
       for {
         record                             <- goodsRecordConnector.getRecord(request.eori, recordId)
+        recordIsLocked                      = record.adviceStatus match {
```

Comment:
Not necessary if you are happy, but if you want to, you could add this as a method on the record itself, and then just call that here

Reply:
can do this as improvement in the other PR.